README.md

```diff
@@ -118,7 +118,9 @@ to do this is to open Electrum's config file and edit the lines to
 `oneserver=true`.
 
 Pro Tip2: run tor on the same machine as Electrum Personal Server. Then by
-default transactions will be broadcast through tor.
+default transactions will be broadcast through tor. If running tor, also set
+`walletbroadcast=0` in your `bitcoin.conf`. This prevents the wallet from
+rebroadcasting transactions from your node if they aren't confirmed quickly.
```

Comment:
Maybe change to ` This prevents the wallet from rebroadcasting transactions without tor.` and leave out the point about transactions not confirming quickly.